Sample/SampleForageSDK/AppDelegate.swift

```diff
@@ -12,13 +12,6 @@ import UIKit
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        ForageSDK.setup(
-            ForageSDK.Config(
-                merchantID: ClientSharedData.shared.merchantID,
-                sessionToken: ClientSharedData.shared.sessionToken
-            )
-        )
```

Comment:
I don't think this block was necessary for our sample app^

Reply:
I removed it because calling `ForageSDK.setup` with empty values for `sessionToken` and `merchantID` was causing the logger and launchdarkly to initialize in the wrong environment